### PR TITLE
The offstation cargo update (Ready)

### PR DESCRIPTION
**Pull request by shellspeed1**
## About The Pull Request

First things first, credit to tmyqlfpir aka Feignsleep for making the initial cargo crate import code. I probably butchered it when I messed with it to make it only import on console so make sure to yell at me if there are issues.

This PR does several things. 

FIRST it brings Tarkon up to date with Interdyne as far as cargo goes. In this case, that means an express cargo console, selling pad, and powerator tied to a budget account.

Second, thing, both Interdyne and Tarkon now have access to cargo company items. Its a bit janky but it should do for at least now. With how it was done, we should be able to add unique items to the consoles for each faction, although that might need some work.

As a minor note, I also nerfed the interdyne powerator. It had a 2x multiplier and I'm not sure that is really needed.

## Why It's Good For The Game

Cargo goes brrrt, also Tarkon could use some love.

## Proof Of Testing

image

## Changelog

:cl:
add: Cargo Companies are now available to offstation factions
add: Tarkon now has a cargo setup available to them
balance: rebalanced something
/:cl:

### CODE REVIEW

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
I forgot to actually submit my review, lol

**Comment by shellspeed1** [comment]
I'll get to it in a couple of days, bonk me if I forget

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Comment by shellspeed1** [comment]
Found a minor issue with existing stuff, will get it figured out and fixed shortly so listings are complete

File: modular_skyrat/modules/tarkon/code/misc-fluff/research.dm [hunk 1]
```diff
@@ -27,7 +27,11 @@
 	design_ids = list(
 		"mod_plating_tarkon",
 		"arcs",
-		"rcd_tarkon"
+		"rcd_tarkon",
+		"powerator_tarkon",
+		"cargoconsole_tarkon",
+		"bountypad_tarkon", //start bubber edit
+		"bountyconsole_tarkon"//end bubber edit
```
Comment (Swiftfeather) [inline modular_skyrat/modules/tarkon/code/misc-fluff/research.dm:34]:
```		
		"rcd_tarkon", //Bubber edit (the comma)
		"powerator_tarkon", //Bubber edit addition start
		"cargoconsole_tarkon",
		"bountypad_tarkon", 
		"bountyconsole_tarkon"//Bubber edit addition end
```

**Review by Swiftfeather — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/tarkon/code/misc-fluff/research.dm [hunk 2]
```diff
@@ -80,7 +84,6 @@
 		RND_CATEGORY_TOOLS + RND_SUBCATEGORY_TOOLS_ENGINEERING_ADVANCED
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING
```
Comment (Swiftfeather) [inline modular_skyrat/modules/tarkon/code/misc-fluff/research.dm:86]:
```suggestion
	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING
	
```

**Comment by Majkl-J** [comment]
I already read through this before and it's good, so

**Comment by Hotmud9** [comment]
I am losing my mind; WHERE ARE THESE BOARDS

**Pull request merged**